app/models/company.rb

```diff
@@ -1,5 +1,30 @@
 class Company < ActiveRecord::Base
+  after_create :get_info
+
   has_many :jobs
 
   validates :name, presence: true
+
+  private
+
+  def get_info
+    result = GlassdoorService.new.company(name)
+    if result
+      update_attributes(url: result["website"],
+                        industry: result["industry"],
+                        logo_url: result["squareLogo"],
+                        overall_rating: result["overallRating"].to_f,
+                        rating_description: result["ratingDescription"],
+                        cultureandvalues_rating: result["cultureAndValuesRating"].to_f,
```

Comment:
How about moving all these over to two spaces to the right of update_attributes.